src/wazuh_modules/vulnerability_scanner/src/scanOrchestrator/packageScanner.hpp

```diff
@@ -57,117 +57,83 @@ class TPackageScanner final : public AbstractHandler<std::shared_ptr<TScanContex
-                !L2Translations.empty())
-            {
-                logDebug2(WM_VULNSCAN_LOGTAG,
-                          "Translation for package '%s' in platform '%s' found in Level 2 cache.",
-                          packageName.c_str(),
-                          osPlatform.c_str());
+            const std::string packageName {Utils::toLowerCase(std::string(data->packageName()))};
+            const std::string packageVendor {Utils::toLowerCase(std::string(data->packageVendor()))};
 
-                // Retrieve translations from cache and initiate vulnerability scans
-                for (const auto& translation : L2Translations)
-                {
-                    logDebug1(WM_VULNSCAN_LOGTAG,
-                              "Initiating a vulnerability scan for package '%s' (%s) (%s) with CVE Numbering "
-                              "Authorities (CNA) '%s' on Agent '%s' (ID: '%s', Version: '%s').",
-                              translation.translatedProduct.c_str(),
-                              data->packageFormat().data(),
-                              translation.translatedVendor.c_str(),
-                              cnaName.c_str(),
-                              data->agentName().data(),
-                              data->agentId().data(),
-                              data->agentVersion().data());
-
-                    PackageData translatedPackage = {.name = translation.translatedProduct,
-                                                     .vendor = translation.translatedVendor,
-                                                     .format = data->packageFormat().data(),
-                                                     .version = data->packageVersion().data()};
-
-                    m_databaseFeedManager->getVulnerabilitiesCandidates(cnaName, translatedPackage, vulnerabilityScan);
-                }
-                // Store translations in Level 1 cache
-                m_translationL1Cache.insertKey(cacheKey, std::move(L2Translations));
-                return true;
-            }
+            logDebug1(WM_VULNSCAN_LOGTAG,
+                      "Initiating a vulnerability scan for package '%s' (%s) (%s) with CVE Numbering "
+                      "Authorities (CNA) "
+                      "'%s' on Agent "
+                      "'%s' (ID: '%s', Version: '%s').",
+                      packageName.c_str(),
+                      data->packageFormat().data(),
+                      packageVendor.c_str(),
+                      cnaName.c_str(),
+                      data->agentName().data(),
+                      data->agentId().data(),
+                      data->agentVersion().data());
+
+            PackageData package = {.name = packageName,
+                                   .vendor = packageVendor,
+                                   .format = data->packageFormat().data(),
+                                   .version = data->packageVersion().data()};
```

Comment:
I think we can reduce the number of variables in this section:
```suggestion
            PackageData package = {.name = Utils::toLowerCase(std::string(data->packageName())),
                                   .vendor = Utils::toLowerCase(std::string(data->packageVendor())),
                                   .format = data->packageFormat().data(),
                                   .version = data->packageVersion().data()};

            logDebug1(WM_VULNSCAN_LOGTAG,
                      "Initiating a vulnerability scan for package '%s' (%s) (%s) with CVE Numbering "
                      "Authorities (CNA) "
                      "'%s' on Agent "
                      "'%s' (ID: '%s', Version: '%s').",
                      package.name.c_str(),
                      data->packageFormat().data(),
                      package.vendor.c_str(),
                      cnaName.c_str(),
                      data->agentName().data(),
                      data->agentId().data(),
                      data->agentVersion().data());
```

Reply:
Fixed!